### ISSUE TITLE
Rewrite current test split as validation

### ISSUE THREAD

**Issue by gacsta** [gacsta/Industrial-Heat-Transfer-Predictor#4]
The portion of the data used for evaluating and fine tuning the model is called validation set. 
Try to split the test portion of the data at the .lvm file concatenating script (Tind_Tindn_Gama.py ).
Plot results from the "true" testing phase.

**Comment by gacsta** [comment]
Try k-fold cross validation method as a replacement for the current train_test_split.
